lib/sms/Kconfig

```diff
@@ -18,6 +18,13 @@ config SMS_SUBSCRIBERS_MAX_CNT
 	help
 	  Maximum number of subscribers that can register to SMS library.
 
+config SMS_DELIVERY_REPORT
+	bool "SMS delivery reports are requested"
+	default y
```

Comment:
I'm tempted to set this disabled by default, which would require an update to migration guide, but I'm checking internally whether that's feasible or not.

Reply:
May need to double-check the wording. 
The param <ds> is for SMS-SUBMIT REPORT, not for SMS-DELIVERY (param <mt>).

Reply:
I had modified this feature as SMS-STATUS_REPORT is what we are subscribing for. SMS-DELIVER-REPORT (not used by SMS lib) is from MS to SMSC and sent for SMS-DELIVER (MT SMS).

Reply:
Status report is nice, see modem trace. And we supress this URC.

```
00:29:23.163773  sms_status_report_ind_t { header : { msg_id : SMS_STATUS_REPORT_IND, sender : TASK_SMS, receiver : TASK_AT } }
00:29:23.163803 L23_AT_STRING AT_STRING <= +CDS: 26
```